web/src/OverviewTableBuildButton.tsx

```diff
@@ -15,8 +20,10 @@ export const OverviewTableBuildButton = styled(BuildButton)`
   &:hover .fillStd {
     fill: ${Color.white};
   }
-  & > svg {
+  & .icon {
```

Comment:
The & > svg does not seem to be matching anything in master, and I failed to come up with a selector that would match it within 15 minutes, so I went with a class.